iroha-cli/interactive/impl/interactive_query_cli.cpp

```diff
@@ -20,8 +20,8 @@
 #include <fstream>
 
 #include "client.hpp"
-#include "crypto/crypto.hpp"
-#include "crypto/hash.hpp"
+#include "cryptography/ed25519_sha3_impl/internal/ed25519_impl.hpp"
```

Comment:
These changes (and similar ones in other files) were done just because of moving crypto from libs to shared_model. It looks like now we use ed25519 with sha-3 crypto directly - and it does but it is absolutely the same as before. It will be improved in separate PR.